hasura/metadata/cron_triggers.yaml

```diff
@@ -1,44 +1 @@
-- name: check-nominee
```

Comment:
don't think this should be removed

Reply:
woo good catch. i had my local state in a funk from seeding issues when i did a metadata export

Reply:
fixed this